spark/src/test/scala/org/apache/comet/CometFuzzTestBase.scala

```diff
@@ -35,12 +35,15 @@ import org.apache.spark.sql.execution.SparkPlan
 import org.apache.spark.sql.execution.adaptive.AdaptiveSparkPlanHelper
 import org.apache.spark.sql.internal.SQLConf
 
-import org.apache.comet.testing.{DataGenOptions, ParquetGenerator, SchemaGenOptions}
+import org.apache.comet.testing.{DataGenOptions, FuzzDataGenerator, ParquetGenerator, SchemaGenOptions}
 
 class CometFuzzTestBase extends CometTestBase with AdaptiveSparkPlanHelper {
 
   var filename: String = null
 
+  /** Filename for data file with deeply nested complex types */
+  var complexTypesFilename: String = null
```

Comment:
would be great to add if this filename is input or output or temp location?

Reply:
I will improve the comments in my next PR later today